### DISCUSSION TITLE
[BUG] узел не синхронизирован с сетью

### DISCUSSION THREAD

**Discussion by Alekcsandr516** [Chia-Network/chia-blockchain#3269] [Support]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS: [e.g. Linux]
- OS Version/Flavor: [e.g. CentOS 7.2]
- CPU: [e.g. Intel Xeon 8175M]

**Additional context**

**Comment by Alekcsandr516** [comment]
продолжает засеивать , но но не синхронизирует , не фармит версия 1.1.1 но ОС Windows

**Comment by ESKCOBAR** [comment]
админы говорят установить новую версию 1.1.2 но если честно и она тупит не синхронизирует у меня уже второй день!

**Comment by angry-mammoth** [comment]
Открываем порт 8444 и будет всё работать :)

**Comment by stalit1** [comment]
Cылка на ютуб https://www.youtube.com/watch?v=f4PS6ut6fQ8

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
